lib/piggly/command/base.rb

```diff
@@ -110,11 +118,11 @@ def o_connection_name(config)
       end
 
       def o_version(config)
-        lambda { puts "piggly #{VERSION} #{VERSION::RELEASE_DATE}"; exit! }
+        lambda {|x| puts "piggly #{VERSION} #{VERSION::RELEASE_DATE}"; exit! }
       end
 
       def o_dry_run(config)
-        lambda { config.dry_run = true }
+        lambda {|x| config.dry_run = true }
       end
```

Comment:
Later versions of Ruby I've found get upset if a lambda is missing parameters even if they aren't used.